compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/Scrollbar.desktop.kt

```diff
@@ -183,7 +185,7 @@ fun HorizontalScrollbar(
 ) = Scrollbar(
     adapter,
     modifier,
-    reverseLayout,
+    reverseLayout xor (LocalLayoutDirection.current == LayoutDirection.Rtl),
```

Comment:
While correct, maybe make more explicit `if (rtl) !reverse else reverse`?

Reply:
Ok, let's obfuscate the code :).

Done